### PR TITLE
POC button

### PR DESCRIPTION
**Pull request by angelozerr**
POC button (please don't merge it)

### CODE REVIEW

File: src/qute/commands/registerCommands.ts [hunk 1]
```diff
@@ -14,6 +14,22 @@ export function registerVSCodeQuteCommands(context: ExtensionContext) {
...
   registerJavaDefinitionCommand(context);
   registerConfigurationUpdateCommand(context);
+  registerVal(context);
+}
+
+function registerVal(context: ExtensionContext) {
+  context.subscriptions.push(commands.registerCommand('qute.validation.enabled.toggle', async (uri?: Uri) => {
+    const templateUri =   uri.toString();
+
+    const result = await commands.executeCommand('qute.command.validation.template.status', templateUri);
```
Comment (AlexXuChen) [inline src/qute/commands/registerCommands.ts:24]: It seems this command doesn't exist. Is this meant to check if  the templateUri exists in the qute.validation.excluded array?

Comment (angelozerr) [reply src/qute/commands/registerCommands.ts:24]: This command exists when qute ls is started. This command gives you information about why the qute template validation is enabled or not.

A qute template validation is enabled if qute.validation.enabled is true and the templateuri doesnt matches one or several patterns from qute.validation.excluded.

A qute template validation is disabled if qute.validation.enabled is set to false or templateuri matches one or several patterns from qute.validation.excluded.

In other words according to the command result you can know if the qute template must be validated or not AND which settings you must update to enable or disable the validation.

**Comment by angelozerr** [comment]
Close for https://github.com/redhat-developer/vscode-quarkus/pull/455
